tests/components/analytics/test_analytics.py

```diff
@@ -73,46 +77,59 @@ async def test_load_with_supervisor_without_diagnostics(hass):
 
 async def test_failed_to_send(hass, caplog, aioclient_mock):
     """Test failed to send payload."""
-    aioclient_mock.post(ANALYTICS_ENDPOINT_URL, status=400)
+    aioclient_mock.post(ANALYTICS_ENDPOINT_URL_DEV, status=400)
     analytics = Analytics(hass)
     await analytics.save_preferences({ATTR_BASE: True})
     assert analytics.preferences[ATTR_BASE]
-    await analytics.send_analytics()
-    assert "Sending analytics failed with statuscode 400" in caplog.text
+
+    with patch(
+        "homeassistant.components.analytics.analytics.HA_VERSION", MOCK_VERSION_DEV
```

Comment:
If we patch it to the production version we don't need to change endpoint aioclient mock in all the existing tests.

Reply:
Does it matter?

Reply:
It's always nice to keep the changeset small. And that we had to change those endpoints was a sign that it would fail on master to me.